configure

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/usr/bin/env bash -e
```

Comment:
I don't think that works as the shebang only supports a single argument. That would pass `bash -e` as executable name to `env`. You'd have to use `set -e` below the shebang.

Reply:
fixed.